src/devices/src/virtio/block/persist.rs

```diff
@@ -16,15 +16,23 @@ use versionize_derive::Versionize;
 use virtio_gen::virtio_blk::VIRTIO_BLK_F_RO;
 use vm_memory::GuestMemoryMmap;
 
-use super::*;
-use crate::virtio::block::device::FileEngineType;
+use crate::virtio::block::device::{Block, CacheType, FileEngineType};
+use crate::virtio::block::io::Error as IoError;
+use crate::virtio::block::{Error, NUM_QUEUES, QUEUE_SIZE};
+use crate::virtio::device::DeviceState;
 use crate::virtio::persist::VirtioDeviceState;
-use crate::virtio::{DeviceState, TYPE_BLOCK};
+use crate::virtio::TYPE_BLOCK;
 
+/// Holds info about block's cache type. Gets saved in snapshot.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, Versionize)]
 // NOTICE: Any changes to this structure require a snapshot version bump.
 pub enum CacheTypeState {
```

Comment:
Nit: Derives are typically written directly on top of the type.
```suggestion
/// Holds info about block's cache type. Gets saved in snapshot.
// NOTICE: Any changes to this structure require a snapshot version bump.
#[derive(Clone, Copy, Debug, PartialEq, Eq, Versionize)]
pub enum CacheTypeState {
```